doc/conf.py

```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'FOSHttpCache'
-copyright = '2014, FriendsOfSymfony'
+copyright = u'2014–2016, FriendsOfSymfony'
```

Comment:
yes, in the php files we have a file header configured, and then we can just change the year in styleci to have it updated everywhere